README.md

```diff
@@ -2,18 +2,35 @@
 
 Calculate the incident radiation on geometry.
 
+Such studies of incident radiation can be used to approximate the energy that can
+be collected from photovoltaic or solar thermal systems. They are also useful
+for evaluating the impact of a building's orientation on both energy use and the
+size/cost of cooling systems. For studies of photovoltaic potential or building
+energy use impact, a sky matrix from EPW radiation should be used. For studies
+of cooling system size/cost, a sky matrix derived from the STAT file's clear sky
+radiation should be used.
+
+NOTE THAT NO REFLECTIONS OF SOLAR ENERGY ARE INCLUDED
```

Comment:
Hey @chriswmackey! Am I right that this is supposed to be ARE NOT INCLUDED?
```suggestion
NOTE THAT NO REFLECTIONS OF SOLAR ENERGY ARE NOT INCLUDED
```

Reply:
Ah, right. Thank you for catching that.

Reply:
Actually, it's correct because it says `NO REFLECTIONS` so adding a `NOT` would be a double negative. But I can still change it to be a little clearer.

Reply:
Ah! I missed the `No` before the reflection!